### PR TITLE
[PT FE]: support aten:resolve_neg, aten::resolve_conj

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/frontends/pytorch/src/op_table.cpp [hunk 1]
```diff
@@ -468,6 +468,11 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::repeat_interleave", op::translate_repeat_interleave},
         {"aten::reshape", op::translate_reshape},
         {"aten::reshape_as", op::translate_reshape_as},
+        // TO DO: enable behaviour for resolve_conj and resolve_neg complex tensors,
+        // when complex dtype will be supported
+        // for real dtypes, these operation return tensor without changes and can be skipped
```
Comment (eaidova) [inline src/frontends/pytorch/src/op_table.cpp:473]:
```suggestion
        // for real dtypes, these operations return input tensor without changes and can be skipped
```

File: tests/layer_tests/pytorch_tests/test_resolve_conj_neg.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+class TestResolveConjNeg(PytorchLayerTest):
+    def _prepare_input(self, dtype="float32"):
+        import numpy as np
+        return (np.random.randn(2, 4).astype(dtype),)
+
+    def _prepare_input_complex(self):
+        import numpy as np
+        return (np.array([[2+3j, 3-2j, 4-9j,10+1j], [1-3j, 3+2j, 4+9j,10-5j]]), )
```
Comment (rkazants) [inline tests/layer_tests/pytorch_tests/test_resolve_conj_neg.py:16]: you can randomly generate imag and real parts and create complex tensor

File: src/frontends/pytorch/src/op_table.cpp [hunk 3]
```diff
@@ -468,6 +468,11 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::repeat_interleave", op::translate_repeat_interleave},
         {"aten::reshape", op::translate_reshape},
         {"aten::reshape_as", op::translate_reshape_as},
+        // TO DO: enable behaviour for resolve_conj and resolve_neg complex tensors,
+        // when complex dtype will be supported
+        // for real dtypes, these operations return input tensor without changes and can be skipped
+        {"aten::resolve_conj", op::skip_node},
+        {"aten::resolve_neg", op::skip_node},
```
Comment (rkazants) [inline src/frontends/pytorch/src/op_table.cpp:475]: Please consider to move `ComplexTypeMark` to common frontend and implement normal translators.
Create JIRA ticket for this to set in `TODO`.
Also, it is a bit strange that the model uses such operations and as if there are no complex tensors.

Comment (eaidova) [reply src/frontends/pytorch/src/op_table.cpp:475]: as it was discussed yesterday with @slyalin I'll not do that for now

**Review by mvafin — APPROVED** [review]

File: src/frontends/pytorch/src/op_table.cpp (hunk 3 above)
Comment (rkazants) [reply src/frontends/pytorch/src/op_table.cpp:475]: Let us create JIRA ticket for that then.

Comment (slyalin) [reply src/frontends/pytorch/src/op_table.cpp:475]: Let's create a ticket and do not block.

**Review by slyalin — APPROVED** [review]

**Pull request merged**